cmd/podman/root.go

```diff
@@ -103,13 +103,22 @@ func persistentPreRunE(cmd *cobra.Command, args []string) error {
 	// TODO: Remove trace statement in podman V2.1
 	logrus.Debugf("Called %s.PersistentPreRunE(%s)", cmd.Name(), strings.Join(os.Args, " "))
 
-	cfg := registry.PodmanConfig()
-
 	// Help is a special case, no need for more setup
 	if cmd.Name() == "help" {
 		return nil
 	}
 
+	cfg := registry.PodmanConfig()
+	if cfg.Remote {
+		if !cmd.Flag("url").Changed {
+			_ = cmd.Flags().Set("url", cfg.Config.Engine.RemoteURI)
```

Comment:
Why not just default to these values, and only use them if user specifies remote.  That way the user will see them if he does podman run --help?

Reply:
D*mn, and I even had a TODO to implement this way.  Fixed.

```
$ bin/podman --help |grep identity                                                    
      --identity string           path to SSH identity file, (CONTAINER_SSHKEY) (default "/home/jhonce/.ssh/ocp-tier3") 
```